node/resolvers/Queries/Settings.ts

```diff
@@ -1,31 +1,27 @@
 import {
     B2B_SETTINGS_DATA_ENTITY,
-    B2B_SETTINGS_FIELDS,
-    B2B_SETTINGS_SCHEMA_VERSION
+    B2B_SETTINGS_FIELDS
 } from '../../mdSchema'
 import GraphQLError from '../../utils/GraphQLError'
 import checkConfig from '../config'
 
 const B2BSettings = {
-    getB2BSettings: async (_: void, { page,
-        pageSize }: {
-            page: number
-            pageSize: number
-        }, ctx: Context) => {
+    getB2BSettings: async (_: void, {id}: {id: string}, ctx: Context) => {
         const {
             clients: { masterdata },
         } = ctx
-
+        
         // create schema if it doesn't exist
         await checkConfig(ctx)
 
         try {
-            return await masterdata.searchDocumentsWithPaginationInfo({
+            const document = await masterdata.getDocument({
                 dataEntity: B2B_SETTINGS_DATA_ENTITY,
                 fields: B2B_SETTINGS_FIELDS,
-                pagination: { page, pageSize },
-                schema: B2B_SETTINGS_SCHEMA_VERSION,
+                id
             })
+
+            return {data: [document]}
```

Comment:
This doesn't need to be wrapped in array. We probably also need to update the return type in the graphql schema.